src/site/navigation/facility_no_drupal_page_sidebar_nav.drupal.liquid

```diff
@@ -5,110 +5,114 @@
 
     For an example of how to make a string look like this, look at in `src/site/layouts/person_profile.drupal.liquid` where sidebarData is being assigned.
 {% endcomment %}
-<nav id="va-detailpage-sidebar" data-drupal-sidebar class="va-c-facility-sidebar usa-width-one-fourth va-sidebarnav">
-    <div>
-        <button class="va-btn-close-icon va-sidebarnav-close" type="button" aria-label="Close this menu"></button>
 
-        {% assign deepLinksObj = sidebarData | jsonToObj %}
-        <div class="left-side-nav-title">
-            <h4>{{ deepLinksObj.link.label }}</h4>
-        </div>
+{% if buildtype != 'vagovprod' %}
```

Comment:
This PR is behind this conditional so that we can merge it in before it actually goes live on production.